MediaBrowser.Api/Playback/Hls/DynamicHlsService.cs

```diff
@@ -939,7 +939,8 @@ protected override string GetCommandLineArguments(string outputPath, EncodingOpt
             if (isEncoding && startNumber > 0)
             {
                 var startTime = state.SegmentLength * startNumber;
-                timeDeltaParam = string.Format("-segment_time_delta -{0}", startTime);
+                string startTimeString = startTime.ToString(_ffmpegTimeDeltaFormat);
+                timeDeltaParam = string.Format("-segment_time_delta {0}", startTimeString);
```

Comment:
The line you changed has a minus sign now, so this will likely break..

Reply:
I have changed things now so it will never be negative. In fact it can't be negative else ffmpeg will throw an error so I really don't know what the - was doing there. Anyways it should be better now. https://github.com/Tthecreator/jellyfin/blob/9ca928af107c4de04862bed2e543ba6cb948244b/MediaBrowser.Api/Playback/Hls/DynamicHlsService.cs#L942-L945

Reply:
Or is this solution to hacky?